src/main/java/FlightsReporter.groovy

```diff
@@ -2,256 +2,138 @@ import java.time.LocalDateTime
 import java.time.temporal.ChronoUnit
 
 class FlightsReporter {
-    final static int[] CAPACITIES = new int[] {80, 120, 150, 180, 240}
+    Flights flights
 
-    static List<Flight> getAllFlightsTo(List<Flight> flights, Airports destination) {
-        flights.findAll { (it.destination == destination) }
+    void printFlights() {
+        flights.printFlights()
     }
 
-    static List<Flight> getAllFlightsFrom(List<Flight> flights, Airports from) {
-        flights.findAll {it.from == from }
+    void printFlightsTo(Airports destination) {
+        flights.printFlightsTo(destination)
     }
 
-    static void printFlights(List<Flight> flights) {
-        flights.each {println(it)}
+    void printFlightsFrom(Airports from) {
+        flights.printFlightsFrom(from)
     }
 
-    static int passengerAmount(List<Flight> flights) {
-        flights.collect {it.passengers}.sum() ?: 0 as int
+    int getPassengerCount() {
+        flights.getPassengerCount()
     }
 
-    static int passengerAmountTo(List<Flight> flights, Airports to) {
-        flights.findAll {it.destination == to }.collect {it.passengers}.sum() ?: 0 as int
+    int getPassengerCountTo(Airports to) {
+        flights.getPassengerCountTo(to)
     }
 
-    static int passengerAmountFrom(List<Flight> flights, Airports from) {
-        flights.findAll {it.from == from }.collect {it.passengers }.sum() ?: 0 as int
+    int passengerAmountFrom(Airports from) {
+        flights.passengerAmountFrom(from)
+    }
+
+    List<Flight> getAllFlightsFrom(Airports from) {
+        flights.getAllFlightsFrom(from)
+    }
+
+    List<Flight> getAllFlightsTo(Airports to) {
+        flights.getAllFlightsTo(to)
```

Comment:
If all is done by the flights class what is the meaning of the FlightsReporter ? feels redundant to me. Try to keep it simple